### PR TITLE
Bug 1163459 - [Gaia-Raptor] Make runners in Suite customizable

### PR DESCRIPTION
**Pull request by snowmantw**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,6 +50,10 @@ var createRunner = function(options, callback) {
     });
 };
 
+var registerRunner = function(phase, path) {
```
Comment (eliperelman) [inline index.js]: Needs JSDoc comments.

File: index.js [hunk 2]
```diff
@@ -68,6 +72,15 @@ var raptor = function(options, callback) {
     options.time = Date.now();
   }
 
+  // If register a new runner: { phase: String, path: ModulePath }.
+  // If register a new runner there is no need to add 'phase' in options.
```
Comment (eliperelman) [inline index.js]: s/register/registering/

File: index.js [hunk 3]
```diff
@@ -68,6 +72,15 @@ var raptor = function(options, callback) {
...
+  if (options.runner) {
+    var phase = options.runner.phase;
+    var path  = options.runner.path;
```
Comment (eliperelman) [inline index.js]: nit: don't line up `=`, move left 1 space.

File: index.js [hunk 4]
```diff
@@ -68,6 +72,15 @@ var raptor = function(options, callback) {
...
+    Suite.registerRunner(phase, path);
```
Comment (eliperelman) [inline index.js]: nit: put line break after variables.

File: index.js [hunk 5]
```diff
@@ -120,4 +133,4 @@ var raptor = function(options, callback) {
   }, complete);
 };
 
-module.exports = raptor;
\ No newline at end of file
+module.exports = raptor;
```
Comment (eliperelman) [inline index.js]: Nothing appears to have changed on this line.

Comment (snowmantw) [inline index.js]: I don't know what's happened here. The GitHub says it has 'no new line at the end of file', which usually appears as an issue caused by the editor. I'm sure I touched nothing about this line.

**Pull request merged**